zScanner/ViewModels/Documents/DocumentViewModel.swift

```diff
@@ -120,6 +126,17 @@ class DocumentViewModel {
             .disposed(by: disposeBag)
     }
     
+    private let realm = try! RealmDatabase()
```

Comment:
We need to talk about benefits of dependency injection :D 
You should not create an instance of the Realm inside of the view model. The view model has the `database` provided on init. You should use it.